liquidctl/driver/corsair_hid_psu.py

```diff
@@ -84,6 +84,10 @@ class CorsairHidPsu(UsbHidDriver):
             'fpowin115': (6.244705156199815e-05,  1.0234738310580973, 15.293509559389241),
             'fpowin230': (5.9413179794350966e-05, 1.0023670927127724, 15.886126793547152),
         }),
+        (0x1b1c, 0x1c23, 'Corsair HX1200i ATX 3.0', {
```

Comment:
Let's go back to ATX 3.1 (see other comment).

Additionally, could you also add comments with the specific PSU model (CP-9020281-NA) and pointing out that the input power curves come from the 80 Plus certification, and thus are based on 115 V input testing (only)?